tests/get_or_create/tests.py

```diff
@@ -503,6 +505,45 @@ def raise_exception():
         )
         self.assertFalse(created)
 
+    def test_mti_update_non_local_concrete_fields(self):
+        journalist = Journalist.objects.create(name="Jane", specialty="Politics")
+        journalist, created = Journalist.objects.update_or_create(
+            pk=journalist.pk,
+            defaults={"name": "John"},
+        )
+        self.assertFalse(created)
```

Comment:
Use `assertIs` here and below instead of `assertFalse`:
```suggestion
        self.assertIs(created, False)
```
(see [Coding style docs](https://docs.djangoproject.com/en/4.1/internals/contributing/writing-code/coding-style/#python-style))

Reply:
Ah good spot thank you, updated :+1: